rest_sensors.cpp

```diff
@@ -1433,17 +1460,34 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                         }
                         if (data.length() > 0)
                         {
-                            if (sendTuyaRequest(task, TaskThermostat , DP_TYPE_ENUM, 0x6a, data))
+                            if (R_GetProductId(sensor) == QLatin1String("Tuya_THD MOES TRV"))
                             {
-                                updated = true;
+                                if (modeSet == "off")
+                                {
+                                    sendTuyaRequest(task, TaskThermostat , DP_TYPE_ENUM, DP_IDENTIFIER_THERMOSTAT_MODE_2, data);
```

Comment:
In other usages of `sendTuyaRequest()` the return value is checked and an error is reported when it fails, shouldn't it be done here as well?

Reply:
done